### ISSUE TITLE
" superclass must be a Class " with RoR 6.0.1

### ISSUE THREAD

**Issue by tchukuchuk** [actindi/act-fluent-logger-rails#51]
Hi,

I have this issue when my server boots :

```
/usr/local/bundle/gems/act-fluent-logger-rails-0.0.4/lib/act-fluent-logger-rails/logger.rb:6:in `<module:ActFluentLoggerRails>': superclass must be a Class (Module given) (TypeError)
     from /usr/local/bundle/gems/act-fluent-logger-rails-0.0.4/lib/act-fluent-logger-rails/logger.rb:4:in `<main>'
     from /usr/local/bundle/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require'
     from /usr/local/bundle/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `block in require_with_bootsnap_lfi'
     from /usr/local/bundle/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
     from /usr/local/bundle/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require_with_bootsnap_lfi'
     from /usr/local/bundle/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
     from /usr/local/bundle/gems/activesupport-6.0.1/lib/active_support/dependencies.rb:325:in `block in require'
     from /usr/local/bundle/gems/activesupport-6.0.1/lib/active_support/dependencies.rb:291:in `load_dependency'
     from /usr/local/bundle/gems/activesupport-6.0.1/lib/active_support/dependencies.rb:325:in `require'
     from /usr/local/bundle/gems/act-fluent-logger-rails-0.0.4/lib/act-fluent-logger-rails.rb:2:in `<main>'
     from /usr/local/bundle/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require'
     from /usr/local/bundle/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `block in require_with_bootsnap_lfi'
     from /usr/local/bundle/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
     from /usr/local/bundle/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require_with_bootsnap_lfi'
     from /usr/local/bundle/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
     from /usr/local/lib/ruby/2.6.0/bundler/runtime.rb:81:in `block (2 levels) in require'
     from /usr/local/lib/ruby/2.6.0/bundler/runtime.rb:76:in `each'
     from /usr/local/lib/ruby/2.6.0/bundler/runtime.rb:76:in `block in require'
     from /usr/local/lib/ruby/2.6.0/bundler/runtime.rb:65:in `each'
     from /usr/local/lib/ruby/2.6.0/bundler/runtime.rb:65:in `require'
     from /usr/local/lib/ruby/2.6.0/bundler.rb:114:in `require'
     from /app/config/application.rb:9:in `<main>'
     from /usr/local/bundle/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require'
     from /usr/local/bundle/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `block in require_with_bootsnap_lfi'
     from /usr/local/bundle/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
     from /usr/local/bundle/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require_with_bootsnap_lfi'
     from /usr/local/bundle/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
     from /usr/local/bundle/gems/activesupport-6.0.1/lib/active_support/dependencies.rb:325:in `block in require'
     from /usr/local/bundle/gems/activesupport-6.0.1/lib/active_support/dependencies.rb:291:in `load_dependency'
     from /usr/local/bundle/gems/activesupport-6.0.1/lib/active_support/dependencies.rb:325:in `require'
     from /usr/local/bundle/gems/railties-6.0.1/lib/rails/commands/server/server_command.rb:141:in `block in perform'
     from /usr/local/bundle/gems/railties-6.0.1/lib/rails/commands/server/server_command.rb:138:in `tap'
     from /usr/local/bundle/gems/railties-6.0.1/lib/rails/commands/server/server_command.rb:138:in `perform'
     from /usr/local/bundle/gems/thor-0.20.3/lib/thor/command.rb:27:in `run'
     from /usr/local/bundle/gems/thor-0.20.3/lib/thor/invocation.rb:126:in `invoke_command'
     from /usr/local/bundle/gems/thor-0.20.3/lib/thor.rb:387:in `dispatch'
     from /usr/local/bundle/gems/railties-6.0.1/lib/rails/command/base.rb:65:in `perform'
     from /usr/local/bundle/gems/railties-6.0.1/lib/rails/command.rb:46:in `invoke'
     from /usr/local/bundle/gems/railties-6.0.1/lib/rails/commands.rb:18:in `<main>'
     from /usr/local/bundle/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require'
     from /usr/local/bundle/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `block in require_with_bootsnap_lfi'
     from /usr/local/bundle/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
     from /usr/local/bundle/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require_with_bootsnap_lfi'
     from /usr/local/bundle/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
     from /usr/local/bundle/gems/activesupport-6.0.1/lib/active_support/dependencies.rb:325:in `block in require'
     from /usr/local/bundle/gems/activesupport-6.0.1/lib/active_support/dependencies.rb:291:in `load_dependency'
     from /usr/local/bundle/gems/activesupport-6.0.1/lib/active_support/dependencies.rb:325:in `require'
     from /app/bin/rails:9:in `<top (required)>'
     from /usr/local/bundle/gems/spring-2.1.0/lib/spring/client/rails.rb:28:in `load'
     from /usr/local/bundle/gems/spring-2.1.0/lib/spring/client/rails.rb:28:in `call'
     from /usr/local/bundle/gems/spring-2.1.0/lib/spring/client/command.rb:7:in `call'
     from /usr/local/bundle/gems/spring-2.1.0/lib/spring/client.rb:30:in `run'
     from /usr/local/bundle/gems/spring-2.1.0/bin/spring:49:in `<top (required)>'
     from /usr/local/bundle/gems/spring-2.1.0/lib/spring/binstub.rb:11:in `load'
     from /usr/local/bundle/gems/spring-2.1.0/lib/spring/binstub.rb:11:in `<top (required)>'
     from /app/bin/spring:15:in `require'
     from /app/bin/spring:15:in `<top (required)>'
     from bin/rails:3:in `load'
     from bin/rails:3:in `<main>'

````
Could you help me ?

Thanks.

**Comment by jiikko** [comment]
I released version 0.6.0 for rails6. Please use this.
https://rubygems.org/gems/act-fluent-logger-rails/versions/0.6.0

**Comment by tchukuchuk** [comment]
Thanks !
